nntrainer/tensor/short_tensor.cpp

```diff
@@ -278,20 +279,7 @@ std::vector<unsigned int> ShortTensor::argmax() const {
   return result;
 }
 
-float ShortTensor::max_abs() const {
-  const uint16_t *data = (uint16_t *)getData();
-  unsigned int idx;
-
-  uint16_t max_val = data[0];
-  for (unsigned int i = 1; i < size(); i += 1) {
-    uint16_t cur_val = (data[i] >= 0) ? data[i] : -1 * data[i];
-    if (cur_val > max_val) {
-      max_val = cur_val;
-    }
-  }
-
-  return max_val;
-}
+float ShortTensor::max_abs() const { return maxValue(); }
 
 float ShortTensor::maxValue() const {
```

Comment:
And just curious, if this tensor is containing 'unsigned' integer, should it be `max_abs()` ? it is already absolute.

Reply:
sorry I didn't quite understand. do you mean why `max_abs()` should be used?

Reply:
Yes. I think can't see any difference between maxValue and maxAbsoluteValue for this tensor

Reply:
You're right! There's no difference between `max_abs()` and `maxValue()` for this tensor class. That's why max_abs() is using maxValue(). The reason why there are `max_abs()` and `maxValue()` is to be compatible with the TensorBase and Tensor class. While max_abs() is currently a pure virtual function, it will get a compile error without implementing `max_abs()` in the ShortTensor class.